client/web/src/site-admin/SiteAdminBackgroundJobsPage.tsx

```diff
@@ -0,0 +1,483 @@
+                      hostName !== undefined ? run.hostName : run.hostName === hostName ? hostName : null,
+                  undefined
+              )
+    const routineIcon =
+        routine.type === 'PERIODIC' ? (
+            <Icon aria-hidden={true} svgPath={mdiCached} />
+        ) : routine.type === 'PERIODIC_WITH_METRICS' ? (
+            <Icon aria-hidden={true} svgPath={mdiNumeric} />
+        ) : routine.type === 'DB_BACKED' ? (
+            <Icon aria-hidden={true} svgPath={mdiDatabase} />
+        ) : routine.type === 'CUSTOM' ? (
+            <Icon aria-hidden={true} svgPath={mdiShape} />
+        ) : (
+            <Icon aria-hidden={true} svgPath={mdiHelp} />
+        )
+
+    const recentRunsReversed = [...routine.recentRuns].reverse()
+
+    const recentRunsTooltipContent = (
+        <div>
+            {commonHostName ? <Text className="mb-0">All on “{commonHostName}”:</Text> : ''}
+            <ul className="pl-4">
+                {recentRunsReversed.map(run => (
+                    <li key={run.at}>
+                        <Text className="mb-0">
+                            {run.errorMessage ? (
+                                <Icon aria-hidden={true} svgPath={mdiAlert} className="text-danger" />
+                            ) : (
+                                ''
+                            )}{' '}
+                            <Timestamp date={new Date(run.at)} noAbout={true} />
+                            {commonHostName ? '' : ` on the host called “${run.hostName}”,`} for{' '}
+                            <span className={getRunDurationTextClass(run.durationMs, routine.intervalMs)}>
+                                {run.durationMs}ms
+                            </span>
+                            .{run.errorMessage ? ` Error: ${run.errorMessage}` : ''}
+                        </Text>
+                    </li>
+                ))}
+            </ul>
+        </div>
+    )
+    const recentRunsWithErrors = routine.recentRuns.filter(run => run.errorMessage)
+
+    return (
+        <div className={styles.routine}>
+            <div className={styles.nameAndDescription}>
+                <Text className="mb-1 ml-4">
+                    <span className="mr-2">
+                        <StartedStoppedIndicator routine={routine} />
+                    </span>
+                    <Tooltip content={routine.type.toLowerCase().replace(/_/g, ' ')} placement="top">
+                        {routineIcon}
+                    </Tooltip>
+                    <span className="ml-2">
+                        <strong>{routine.name}</strong>
+                    </span>
+                    <span className="ml-2 text-muted">{routine.description}</span>
+                </Text>
+                <Text className="mb-0 ml-4 text-muted">
+                    {routine.intervalMs ? (
+                        <>
+                            {routine.type !== 'DB_BACKED' ? 'Runs ' : 'Checks queue '}every{' '}
```

Comment:
I'm just a positive person. 😛

```suggestion
                            {routine.type === 'DB_BACKED' ? 'Checks queue ' : 'Runs '}every{' '}
```